index.src.html

```diff
@@ -655,6 +655,27 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   with W3C HTML5, those would need to be copied from [[HTML]].
 </section>
 
+<section class="informative">
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  <em>This section is not normative.</em>
```

Comment:
I think the intention is more like, "This section is normative but we want it to go away when CSS gets its act together and specifies all this more precisely."